packages/components/src/modifiers/hds-code-editor/linters/json-linter.ts

```diff
@@ -141,7 +141,7 @@ export default async function jsonLinter(
       }
     });
 
-    onLint?.(diagnostics);
+    onLint?.(diagnostics, editor.state.doc.toString(), editor);
```

Comment:
```suggestion
    onLint?.(diagnostics, doc.toString(), editor);
```

Just a small suggestion bc you already have the value stored as a variable.